src/Storages/MergeTree/IMergeTreeReader.cpp

```diff
@@ -278,4 +278,13 @@ void IMergeTreeReader::checkNumberOfColumns(size_t num_columns_to_read) const
                         "Expected {}, got {}", requested_columns.size(), num_columns_to_read);
 }
 
+String IMergeTreeReader::getMessageForDiagnosticOfBrokenPart(size_t from_mark, size_t max_rows_to_read) const
+{
+    return fmt::format(
```

Comment:
new message did not contain disk type/name but old message did. Is it ok?

Reply:
No, it's was accidentally removed because of wrong conflicts resolution.